go-controller/pkg/ovn/gateway_cleanup.go

```diff
@@ -109,6 +110,32 @@ func (oc *Controller) staticRouteCleanup(nextHops []net.IP) {
 	}
 }
 
+// policyRouteCleanup cleansup all policies on cluster router that have a nextHop
+// in the provided list.
+// - if the LRP exists and has the len(nexthops) > 1: it removes
+// the specified gatewayRouterIP from nexthops
+// - if the LRP exists and has the len(nexthops) == 1: it removes
+// the LRP completely
+func (oc *Controller) policyRouteCleanup(nextHops []net.IP) {
+	for _, nextHop := range nextHops {
+		gwIP := nextHop.String()
+		policyPred := func(item *nbdb.LogicalRouterPolicy) bool {
+			hasGatewayRouterIPNexthop := false
+			for _, nexthop := range item.Nexthops {
+				if nexthop == gwIP {
+					hasGatewayRouterIPNexthop = true
```

Comment:
can't we just do `return true` here instead of using `hasGatewayRouterIPNexthop`?

Reply:
To Surya's defense, this variable predates her change here. :)
Yes, in this case, returning instead of using variable is a good approach.

Reply:
yes agree to both of you :D will update this part and make it cleaner with return true.